assets/css/style.pcss

```diff
@@ -15,33 +15,47 @@
 	display: flex;
 	align-items: center;
 	justify-content: center;
+	width: var(--tribe-play-button-size);
+	height: var(--tribe-play-button-size);
```

Comment:
❓ Rather than fixed  desktop & mobile sizes, could we use clamp to apply fluid scaling to th ebutton size depending on the size (width) of the embed?

Reply:
@dpellenwood I updated this to use container query clamp values. Care to take another look to see if that is what you were looking for?

Reply:
Love it! How does the scaling feel in the browser? As long as it's not janky or weird looking, this is a simpler, more fluid solution, imho.

Reply:
We have transitions on the resizing so it feels pretty smooth. Although, I'm tempted to remove the transitions on the sizing and only apply it to the hover opacity effect since transitions only apply when you stop resizing. This could make the button pretty big if you're going from a large screen width to a small screen width. Usually clamp values are pretty good on text and we're using that everywhere on our Moose sites. I don't see why it would be any different here.